helm/prometheus-rules/templates/alerting-rules/linkerd.deployment.rules.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: linkerd.deployment.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: linkerd-deployment
+    rules:
+    - alert: LinkerdDeploymentNotSatisfied
+      annotations:
+        description: '{{`Linkerd Deployment {{ $labels.namespace}}/{{ $labels.deployment }} is not satisfied.`}}'
+        opsrecipe: managed-app-linkerd/
+      expr: managed_app_deployment_status_replicas_unavailable{deployment=~"linkerd.*"} > 0
+      for: 30m
```

Comment:
This means the alert has to match for 30min before it fires.
The data starts erroring at T+15
The unit test expects it to fire at T+25

Does not add up.

Reply:
Thanks, pushed [4d6cd82](https://github.com/giantswarm/prometheus-rules/pull/689/commits/4d6cd82633d0958f76cb5d6b05730d4caacaf8e3) before reading this